filebeat/tests/system/test_registrar.py

```diff
@@ -1396,3 +1398,8 @@ def test_registrar_files_with_prospector_level_processors(self):
                 "inode": stat.st_ino,
                 "device": stat.st_dev,
             }, file_state_os)
+
+
+if __name__ == '__main__':
```

Comment:
With this addition, can you directly "run" the file without prepending it with nosetests?